tests/scripts/analyze_outcomes.py

```diff
@@ -440,29 +440,17 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
             'component_ref': 'test_tfm_config',
             'component_driver': 'test_tfm_config_p256m_driver_accel_ec',
             'ignored_suites': [
-                # Ignore test suites for the modules that are disabled in the
-                # accelerated test case.
-                'asn1parse',
-                'asn1write',
-                'ecp',
-                'ecdsa',
-                'ecdh',
-                'ecjpake',
-                'bignum_core',
-                'bignum_random',
-                'bignum_mod',
-                'bignum_mod_raw',
-                'bignum.generated',
-                'bignum.misc',
+                # Modules replaced by drivers
+                'asn1parse', 'asn1write',
```

Comment:
In `development`, only some specific test cases in `asn1parse` and `asn1write` are ignored. This pull request is going back to ignoring the whole test suites. It looks like a mistake when resolving conflicts in 57790967534f41f1266255ccd5dd8b7acb5e302a.

I did the merge locally and ended up with the following diff compared to 57790967534f41f1266255ccd5dd8b7acb5e302a:
```
diff --git a/tests/scripts/analyze_outcomes.py b/tests/scripts/analyze_outcomes.py
index 80b6459cd7..a070b01639 100755
--- a/tests/scripts/analyze_outcomes.py
+++ b/tests/scripts/analyze_outcomes.py
@@ -441,6 +441,7 @@ KNOWN_TASKS = {
             'component_driver': 'test_tfm_config_p256m_driver_accel_ec',
             'ignored_suites': [
                 # Modules replaced by drivers
+                'asn1parse', 'asn1write',
                 'ecp', 'ecdsa', 'ecdh', 'ecjpake',
                 'bignum_core', 'bignum_random', 'bignum_mod', 'bignum_mod_raw',
                 'bignum.generated', 'bignum.misc',
@@ -450,12 +451,6 @@ KNOWN_TASKS = {
                 'test_suite_random': [
                     'PSA classic wrapper: ECDSA signature (SECP256R1)',
                 ],
-                'test_suite_asn1parse': [
-                    'INTEGER too large for mpi',
-                ],
-                'test_suite_asn1write': [
-                    re.compile(r'ASN.1 Write mpi.*'),
-                ],
             }
         }
     }

```

Reply:
Oh, I see, that's deliberate. Never mind then. Sorry for the noise.